### PR TITLE
RNA loading UI

### PR DESCRIPTION
**Pull request by hanars**
This modifies the existing UI for for uploading RNA seq outlier data and allows it to support loading TPM from the browser as well. This makes loading RNA-seq data to seqr substantially easier as it no longer will require manually copying files and running manage commands.
Big changes are:
- Updates the UI to require a dropdown to select whether you are loading tpm or outlier data
- Reads the input file from google storage instead of uploading it in the UI, to prevent errors for large files (fixes issue for current outlier upload)
- Moves utility functions, especially those for tpm data, into dataset_utils instead of using a mix of the data manager api file and the manage commands

### CODE REVIEW

File: seqr/views/utils/dataset_utils.py
Comment (bw2) [inline seqr/views/utils/dataset_utils.py:309]: Instead of having a hard-coded list of tissues here, it may be worth treating tissue-type as an opaque label, or even better - allowing any GTEx tissue and adding docs so people uploading data know this. GTEx metadata in this file 
https://storage.googleapis.com/gtex_analysis_v8/annotations/GTEx_Analysis_v8_Annotations_SampleAttributesDS.txt
has an SMTS column with tissue labels for all GTEx samples:

```
   1327 Adipose Tissue
    275 Adrenal Gland
     21 Bladder
   3480 Blood
   1473 Blood Vessel
    217 Bone Marrow
   3326 Brain
    480 Breast
     19 Cervix Uteri
    821 Colon
   1582 Esophagus
      9 Fallopian Tube
   1141 Heart
    104 Kidney
    251 Liver
    867 Lung
   1132 Muscle
    722 Nerve
    195 Ovary
    360 Pancreas
    301 Pituitary
    262 Prostate
    181 Salivary Gland
   2014 Skin
    193 Small Intestine
    260 Spleen
    381 Stomach
    406 Testis
    812 Thyroid
    166 Uterus
    173 Vagina
```

File: seqr/views/utils/dataset_utils.py
Comment (bw2) [inline seqr/views/utils/dataset_utils.py:379]: style - I find `missing_cols = set(expected_columns) - set(header)` easier to read.

**Review by bw2 — CHANGES_REQUESTED** [review]

File: seqr/management/commands/load_rna_seq_tpm.py
Comment (hanars) [reply seqr/management/commands/load_rna_seq_tpm.py:31]: thats the default in python 3 so no need to explicitly set the mode here

File: seqr/views/utils/dataset_utils.py
Comment (hanars) [reply seqr/views/utils/dataset_utils.py:323]: So thats a change in behavior from what we have been doing up until now, which is the behavior based on the requirements as I understood them from when this data was initially added to seqr. I would want to reload all the data if that is thew approach we are going to take, not just add a small number of incremental samples you currently have provided me.

I think this change is out of scope for the current work in this PR. If you think its important, please feel free to make a follow up ticket and provide tpm data for all samples currently in seqr plus the new ones I am adding as part of this ticket

File: seqr/views/utils/dataset_utils.py
Comment (hanars) [reply seqr/views/utils/dataset_utils.py:324]: the UI is not designed for that so its out of scope to change this behavior, and would need us to rethink the database models. If we ever get multiple tissues for a single sample, we can explicitly change this code along with the relevant UI code

File: seqr/views/utils/dataset_utils.py
Comment (hanars) [reply seqr/views/utils/dataset_utils.py:309]: This behavioris perfectly sufficient for our current use case so I see no need to change it now, especially given that the tissue type is a database enumeration which would require migrations to change. If in the future we get new tissue types and need to update the code, it may be worth removing the database enumeration and using this approach instead, but I would want to make that decision when it was actually relevent

File: seqr/views/utils/dataset_utils.py
Comment (bw2) [reply seqr/views/utils/dataset_utils.py:323]: I'm happy to reload all the data after this PR is merged, but can't do it until this is changed.

Comment (hanars) [reply seqr/views/utils/dataset_utils.py:323]: And like I said, this change is out of scope for this PR and I am happy to make this change and reload all the data as part of a different ticket, but I am not willing to block this PR on this change. This has been the behavior for months now, any changes to this behavior are out of scope of simply adding new samples. If you want me to change this behavior please make another ticket, and please review the functionality of the PR as it is scoped in the ticket

**Review by bw2 — APPROVED** [review]

**Pull request merged**
